package.json

```diff
@@ -1,12 +1,29 @@
 {
-  "name": "lakenona-permits",
+  "name": "summerdale-permits",
+  "description": "Simple website to track home status for the Summerdale community.",
   "version": "0.1.0",
-  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/agnelnieves/summerdale-permits"
+  },
+  "contributors": [
+    "Agnel Nieves @agnelnieves",
+    "Fabio M. Costa @fabiomcosta"
+  ],
+  "private": false,
+  "engines": {
+    "node": ">=16.0.0",
+    "yarn": ">=1.22.0",
+    "npm": "please-use-yarn"
```

Comment:
Prevents any contributor from using `npm` instead of `yarn`. This is mainly to prevent having colliding package.lock/yarn.lock files. But tbh im open to either npm or yarn.

 Will throw an error message to the console like the following:

<img width="1321" alt="image" src="https://user-images.githubusercontent.com/5799157/226085825-a181cc8d-c908-482e-8a58-68894b5ed5fb.png">